osh.spec

```diff
@@ -234,15 +234,19 @@ fi
 %{_sbindir}/osh-retention
 %{_sbindir}/osh-stats
 %{_sysconfdir}/osh/hub
+%exclude %{python3_sitelib}/osh/hub/scripts/osh-xmlrpc-client.py
+%exclude %{python3_sitelib}/osh/hub/scripts/umb-emit.py
 %{python3_sitelib}/osh/hub
 %{_unitdir}/osh-retention.*
 %{_unitdir}/osh-stats.*
 %exclude %{python3_sitelib}/osh/hub/settings_local.py*
 %exclude %{python3_sitelib}/osh/hub/__pycache__/settings_local.*
+# These file should be readable and writable by respective groups.
 %dir %attr(775,root,root) %{_localstatedir}/log/osh
```

Comment:
The mode needs to be `755` to avoid conflicts with `osh-worker`:
```
Error: Transaction test error:
  file /var/log/osh conflicts between attempted installs of osh-worker-1.0.0.20240423.145427.g5c09e40.pr_261-1.el9.noarch and osh-hub-1.0.0.20240423.145427.g5c09e40.pr_261-1.el9.noarch
```